lib/gollum.rb

```diff
@@ -14,6 +15,9 @@
 module Gollum
   VERSION = '5.1.2'
 
+  I18n.available_locales = [:en]
+  I18n.load_path = Dir[File.expand_path("lib/gollum/locales") + "/*.yml"]
```

Comment:
We prefer to namespace common built-in classes with `::`, so `::Dir` and `::File` respectively, just so they will not inadvertently clash with classes from external dependencies with the same name.

Reply:
`I18n` is actually the external dependency. But I've added the `::` prefix regardless. 👍